docs/TuxStores.md

```diff
@@ -0,0 +1,172 @@
+The `Stores.createStore` method returns an object which is described below.
+
+###<a id="store-emitChange"></a>Method `store.emitChange` - type: FUNCTION [#](#store-emitChange)
+A store instance is able to invoke the `emitChange` method, which will emit a change event string for any listeners that are listening for that particular string. The change event can be either the default event string `CHANGE` or one provided by the user.
+
+```
+    store.emitChange(); // will emit the default change event type string: 'CHANGE'
+```
+
+#####<a id="emitChange-CHANGE_EVENT"></a>Parameter - `CHANGE_EVENT` - type: STRING - optional [#](#emitChange-CHANGE_EVENT)
+Change event type string that will be broadcast for any listeners that are listening for that event type. If no `CHANGE_EVENT` is specified the default value of `CHANGE` will be used.
+
+```
+    var newChangeEvent = 'NEW_CHANGE_EVENT';
+    store.emitChange(newChangeEvent);
+```
+
+###<a id="store-addChangeListener"></a>Method `store.addChangeListener` - type: FUNCTION [#](#store-addChangeListener)
+A store instance is able to invoke the `addChangeListener` method, which will invoke the provided callback function when the specified event type string is emitted. The change event can be either the default event string `CHANGE` or one provided by the user.
+
+```
+    store.addChangeListener(callback, CHANGE_EVENT);
+```
+
+#####<a id="addChangeListener-callback"></a>Parameter - `callback` - type: FUNCTION - required [#](#addChangeListener-callback)
+The callback function that will be called when the change event type string that is being listened for with this invocation of addChangeListener is broadcast.
+
+```
+    var listenerCallback = function () {
+      this.setState({ messages: MessageStore.all() });
+    };
+    store.addChangeListener(listenerCallback);
+```
+
+#####<a id="addChangeListener-CHANGE_EVENT"></a>Parameter - `CHANGE_EVENT` - type: STRING - optional [#](#addChangeListener-CHANGE_EVENT)
+Change event type string that will be broadcast for any listeners that are listening for that event type. If no `CHANGE_EVENT` is specified the default value of `CHANGE` will be used.
+
+```
+    var listenerCallback = function () {
+      this.setState({ messages: MessageStore.all() });
+    };
+    var newChangeEvent = 'NEW_CHANGE_EVENT';
+    store.addChangeListener(listenerCallback, newChangeEvent);
+```
+
+
+###<a id="store-removeChangeListener"></a>Method `store.removeChangeListener` - type: FUNCTION [#](#store-removeChangeListener)
+A store instance is able to invoke the `removeChangeListener` method, which will remove specified event type string and callback combination from the store. When removing a change listener ensure that the callback and CHANGE_EVENT input parameters are the same values that were submitted when adding the change listener. The change event can be either the default event string `CHANGE` or one provided by the user.
```

Comment:
Change paragraph to: Removes the passed in listener callback from the passed in event string.  If no event is specified than the default `CHANGE` event is used.